db/dcrsqlite/apisource.go

```diff
@@ -1090,7 +1090,8 @@ func (db *wiredDB) GetExplorerAddress(address string, count, offset int64) *expl
 			return nil
 		}
 		return &explorer.AddressInfo{
-			Address: address,
+			Address:  address,
+			MaxLimit: maxcount,
```

Comment:
I'd like for us to start using more descriptive variable names. E.g. what is this a limit on? Also, how are Max and limit not redundant?  I'm guilty of this too btw

Reply:
Yes, limit and maxlimit look redundant but limits value changes from the template not Maxlimit which is always the upper limit of transactions that's can be possible in litemode

Reply:
The key word there is "transactions".  What is it a limit on?